doc/1/controllers/collection/create/index.md

```diff
@@ -0,0 +1,92 @@
+---
```

Comment:
The `collection:create` action does not takes a `mappings` anymore but rather a definition which contain both mappings and settings

See https://docs.kuzzle.io/sdk/js/7/controllers/collection/create/

Also we should always use the plural form: `mappings`